chapters/unitexpressions.tex

```diff
@@ -62,9 +62,10 @@ \section{The Syntax of Unit Expressions}\label{the-syntax-of-unit-expressions}
 \end{lstlisting}
 
 A \lstinline[language=grammar]!unit-symbol! is a non-empty string of \lstinline[language=grammar]!unit-char! characters.
-The set of allowed characters is currently not defined.
+The set of allowed characters is currently not defined, but it is recommended to only use \lstinline[language=grammar]!NON-DIGIT!.
 It is required that basic and derived units of the SI system are recognized, but tools are allowed to additionally support user-defined unit symbols.
 The required unit symbols do not make use of Greek letters, but a unit such as $\Omega$ is spelled out as \lstinline!"Ohm"!.
+Similarly degree is spelled out as \lstinline!deg! for angles, and \lstinline!degC!, \lstinline!degF! and \lstinline!degRk! for temperatures (Celsius, Fahrenheit and Rankine).
```

Comment:
It seems like overspecification to restrict this to angles and temperature.  Shouldn't it be a statement just about symbols, similarly to `"Ohm"`?

Reply:
There are also missing string quotes:
```suggestion
Similarly degree is spelled out as \lstinline!"deg"! for angles, and \lstinline!"degC"!, \lstinline!"degF"! and \lstinline!"degRk"! for temperatures (Celsius, Fahrenheit and Rankine).
```

Reply:
I tried to explain that it is more generic for degree. Good?